### PR TITLE
Price resolver test

### PR DESCRIPTION
**Pull request by Mikearaya**
#322

### CODE REVIEW

**Comment by Mikearaya** [comment]
@pozylon i found some errors related with `_id` not set on the newly added price revolvers and I've fixed it on this PR

**Comment by pozylon** [comment]
When you're done with the tests and Codeship runs through you can merge right away

File: packages/core-products/db/products/helpers.js [hunk 1]
```diff
@@ -60,12 +60,38 @@ const getPriceRange = (prices) => {
 
   return {
     minPrice: {
+      _id: crypto
+        .createHash('sha256')
+        .update(
+          [
+            this._id,
+            Math.random(),
```
Comment (pozylon) [inline packages/core-products/db/products/helpers.js:68]: If you use Math.random here, there is no need for any hash creation. The hashing is thought to make the _id stay the same when the price does not change

**Review by pozylon — CHANGES_REQUESTED** [review]

File: packages/core-products/db/products/helpers.js (hunk 1 above)
Comment (Mikearaya) [reply packages/core-products/db/products/helpers.js:68]: @pozylon id for min and max price will be the same if i don't find a differentiator. that's why i used it

Comment (pozylon) [reply packages/core-products/db/products/helpers.js:68]: Then use "min" and "max" please. Math.random should never be part of an _id hash

**Pull request merged**